### ISSUE TITLE
Multiple Firebase accounts within the same react native app.

### ISSUE THREAD

**Issue by beingshashi** [invertase/react-native-firebase#743]
Hi,
I have a ReactNative BasedApp. I have used google account "A" for FCM , which is working.
And until now , all firebase automatically moving to account "A".

Now we have requirement that , we want to show Firebase Analytics, Events to Account "B".

So we have used `initialiseApp` function. But has got not success.

I have got warning that , the app name which i added inside `initialiseApp` function is already exists which is true.

So basically how do I use particualar App from Account "B" and that too only for analytics and auth and not notifications.

Thank you

**Comment by chrisbianca** [comment]
@beingshashi Analytics and FCM only work with the default App set up using your `.plist` or `.json` configuration file.  This is a restriction of the underlying SDKs.

In the future, for questions, please use our Discord chat: https://discord.gg/t6bdqMs or Stack Overflow: https://stackoverflow.com/questions/tagged/react-native-firebase
